test/performance/odp_sched_perf.c

```diff
@@ -294,7 +320,7 @@ static int destroy_queues(test_global_t *global)
 	while ((ev = odp_schedule(NULL, wait)) != ODP_EVENT_INVALID)
 		odp_event_free(ev);
 
-	for (i = 0; i < ODP_THREAD_COUNT_MAX; i++) {
+	for (i = 0; i < MAX_QUEUES; i++) {
```

Comment:
Same question here. `tot_queue` seems more efficient than `MAX_QUEUES`.

Reply:
changed to tot_queue in v2.